src/app/cars/car-detail/car-detail.component.html

```diff
@@ -0,0 +1,18 @@
+<div *ngIf="car">
```

Comment:
I guess this ngIf is to avoid having a null in car while is loading which could throw errors. It would be better to do an optional evaluation (i.e. car?.id)

Reply:
Changed to an optional evaluation